packages/gestalt/src/DropdownItem.js

```diff
@@ -13,7 +14,7 @@ type Props = {|
   /**
    * When supplied, will display a [Badge](https://gestalt.pinterest.systems/badge) next to the item's label. See the [Badges](https://gestalt.pinterest.systems/dropdown#Badges) variant to learn more.
    */
-  badgeText?: string,
+  badge?: BadgeType,
```

Comment:
Main change 1/3

Reply:
This type will need to be inlined everywhere it's used, as the codegen tool can't pick it up:
<img width="376" alt="Screen Shot 2022-03-15 at 12 11 06 PM" src="https://user-images.githubusercontent.com/12059539/158453576-63c84202-bbbc-4401-b02d-21afd99e5a1d.png">